contracts/multisig/src/types.rs

```diff
@@ -30,16 +30,16 @@ impl MsgToSign {
 }
 
 #[cw_serde]
-pub struct KeyID {
+pub struct WorkerSetID {
```

Comment:
We shouldn't need this struct. WorkerSet already has an `id()` method that returns an id